### ISSUE TITLE
Setting font size for list symbol

### ISSUE THREAD

**Issue by tmpmachine** [EvotecIT/OfficeIMO#201]
Related PR : https://github.com/EvotecIT/OfficeIMO/pull/196

This PR adds option to set font size on list symbol by appending `NumberingSymbolRunProperties` on each `Level`. The issue is that this font size affect all lists with the same `WordListStyle`.

Here I have two list with `Headings111Shifted` style :

```c#
 var ListItems1 = new List<string>() { "sample " }; 
 var ListItems2 = new List<string>() { "gogo " }; 

 {
     int listFontSize = 13;
     int listSymbolFontSize = 24; // applied to all
     var wordList2 = doc.AddList(WordListStyle.Headings111Shifted, false, listSymbolFontSize);
     foreach (var point in ListItems1) {
         wordList2.AddItem($"{point}", 0)
             .SetFontSize(listFontSize);
     }
 }
 {
     int listFontSize = 22;
     int listSymbolFontSize = 55; // not applied
     var wordList2 = doc.AddList(WordListStyle.Headings111Shifted, false, listSymbolFontSize);
     foreach (var point in ListItems2 ) {
         wordList2.AddItem($"{point}", 0)
             .SetFontSize(listFontSize);
     }
 }
```

![image](https://github.com/EvotecIT/OfficeIMO/assets/18110223/29bb22c7-711d-4896-a66d-dc34bac5b0f9)

# Findings so far

Both list correctly reference it's own `NumberingInstance` with it's own `NumberingSymbolRunProperties`.  What could be missing?

![image](https://github.com/EvotecIT/OfficeIMO/assets/18110223/379e770a-6cd6-4e96-89ae-e2d580f45e85)

![image](https://github.com/EvotecIT/OfficeIMO/assets/18110223/7a56ec33-5b9d-42ef-a30f-791165dfbf8b)

![image](https://github.com/EvotecIT/OfficeIMO/assets/18110223/e09067f4-ed6e-4367-b5ae-f3e75946e860)

![image](https://github.com/EvotecIT/OfficeIMO/assets/18110223/30eff4ce-a5f4-4a34-b5d7-6515d76f24c1)

**Comment by PrzemyslawKlys** [comment]
Lists are complicated beasts. I'll take a look. For now I'll link to old PR maybe it will give you something:
- https://github.com/EvotecIT/OfficeIMO/pull/110

**Comment by PrzemyslawKlys** [comment]
So when I look at how Word deals with font size it seems to be applying it to ParagraphProperties

![image](https://github.com/EvotecIT/OfficeIMO/assets/15063294/d83a106c-1648-403a-a870-8037d62c4b73)

And not to AbstractNum. Is there a reason you want it done that way?

**Comment by PrzemyslawKlys** [comment]
I found a reason why it doesn't work. It's related to Nsid. 

![image](https://github.com/EvotecIT/OfficeIMO/assets/15063294/d4216a7e-c9e5-43a4-899a-c55643b80995)

In the original code each list has different Nsid, but only if the list is different. If the list is the same it uses same Nsid (as it's hardcoded). 

For this to work, it needs to be fixed so that Nsid is randomized or have proper numbering like NumberingInstance/AbstractNum. 

I did quick hack and it works...

**Comment by tmpmachine** [comment]
It looks like randomizing `nsid` causing the `continueNumbering` to fail. Created a quick test in db169830e223ce975b5e5fb0ccf717fa5653c094. 

Need more investigation, will check later.

```c#
[Fact]
public void Test_CreatingWordDocumentWithLists3() {
    
    var filePath = Path.Combine(_directoryWithFiles, "CreatedDocumentWithLists2.docx");
    using (var doc = WordDocument.Create(filePath)) {


        doc.AddParagraph("Capybaras:");
        { 
            WordList wl = doc.AddList(WordListStyle.HeadingIA1);
            wl.AddItem("Pablo");
        }
        {
            WordList wl = doc.AddList(WordListStyle.HeadingIA1, true);
            wl.AddItem("Ponyo");
        }

        doc.Save(true);
    }

}
```

![image](https://github.com/EvotecIT/OfficeIMO/assets/18110223/3906899d-af03-4b07-855f-876df2d7fe9a)

**Comment by PrzemyslawKlys** [comment]
I guess when we want to continuenumbering we will now need to refer to the original AbstractNum, instead of adding a new one...

**Comment by tmpmachine** [comment]
> So when I look at how Word deals with font size it seems to be applying it to ParagraphProperties
> 
> And not to AbstractNum. Is there a reason you want it done that way?

Not really, at that time I went straight to abstractnum instead of checking on a higher level (paragraph), so..

.. for future reference, anyone looking a way to customize list symbol can directly append run properties as we wait for more updates:
```c#
WordList wl = doc.AddList(WordListStyle.Headings111Shifted);
var para = wl.AddItem("Pablo").SetBold();

// set bold & font size 43 to list symbol
var paragraphProperties = para._paragraphProperties;
paragraphProperties.Append(
    new RunProperties() {
        Bold = new Bold(),
        FontSize = new FontSize() { 
            Val = "43"
        },
    }
);
```

**Comment by PrzemyslawKlys** [comment]
I've now removed restart numbering/continue numbering as it makes no sense:
- https://github.com/EvotecIT/OfficeIMO/issues/204

If you feel this is somehow wrong please let me know.
